ext/dba/libinifile/inifile.c

```diff
@@ -453,12 +439,12 @@ static int inifile_filter(inifile *dba, inifile *from, const key_type *key, bool
 /* }}} */
 
 /* {{{ inifile_delete_replace_append */
-static int inifile_delete_replace_append(inifile *dba, const key_type *key, const val_type *value, int append, bool *found)
+static int inifile_delete_replace_append(inifile *dba, const key_type *key, const val_type *value, bool is_append, bool *found)
 {
-	size_t pos_grp_start=0, pos_grp_next;
+	size_t pos_grp_start = 0, pos_grp_next;
 	inifile *ini_tmp = NULL;
 	php_stream *fp_tmp = NULL;
-	int ret;
+	int ret = FAILURE;
```

Comment:
wait shouldn't this variable (and the function's return value) be zend_result?

Reply:
Yes, it should, but I feel ideally this should just return `bool`, but I would have needed to go and change stuff at the DBA driver level which I wanted to do in a follow-up.